packages/dd-trace/src/instrumenter.js

```diff
@@ -108,6 +111,8 @@ class Instrumenter {
       log.error(e)
       this.unload(plugin)
       log.debug(`Error while trying to patch ${meta.name}. The plugin has been disabled.`)
+
+      platform.metrics().increment(`datadog.tracer.node.plugin.errors`, true)
```

Comment:
`s/increment/boolean/`?

Reply:
This is an error counter. In general it will be just 1, but if it's higher it means we attempt to patch multiple times on error which is bad.

Reply:
Why `true`?

Reply:
The last parameter is a flag to set it as a monotonic counter instead of gauge counter.

Reply:
oh, weird